rules-topics/08_magic-skills.md

```diff
@@ -523,19 +534,22 @@ In order to cast a spell, you must possess the appropriate Mastery skill, and pu
+| **Duration**     | One Minute |
+| **Incant**     | "By the spirits, I grant you a resurrection." |
+| **Description** |  This spell takes 1 minutes to cast and requires an untethered spirit. At the beginning of the spell, the target must clearly acknowledge the acceptance of the spell, if they do not, the spell fails to no effect. Once the spell begins, the target's body begins to immediately reform, incurring strain as per a normal resurrection. The caster must be in concentration and unimpeded for the entire casting time of this spell, else the spell will fail. If the caster’s concentration is interrupted, or the reforming body is affected by any detrimental effect, the spell fails and the target immediately suffers a second death. Prior to the casting time the player of the caster, and the player of the untethered spirit, are to verify that the spirit does not have too much strain to resurrect. A marshal should also be present to oversee the process. Once the Resurrection spell is complete, the spirit instantly reforms its body next to the spell caster. The body appears with a full health pool, and all skills renewed as if they had completed a long rest.  This is a LIFE effect. |
 
-#### **Dispel Magic**
-| **Spell**            | **Dispel Magic** |
-|:---|:---|
-| **Duration**     | Instant |
-| **Incant**     | "By the spirits, I dispel magic." |
-| **Description** |  This spell will remove all spell effects from the target character or item. This spell does not remove effects caused by toxins or physical sources. Dispel Magic is assumed to remove a spell effect unless otherwise noted in the spell description. |
+### Spirit Rank X
+#### Death
+| **Spell**    | **Death**                                                |
+|--------------|----------------------------------------------------------|
+| **School**   | Spirit                                                   |
+| **Level**    | 10                                                       |
+| **Duration** | Instant                                                  |
+| **Incant**   | "I grant you the gift of death X... Magic Heal Wounds X" |
+| **Description** | The caster saps the vital force of the target for 25 damage, then within 3 seconds, the caster may touchcast a Magic Heal Wounds 25 _or_ Magic Necrotic Touch 25. The damage and healing can be increased by +5 points for each additional mana point expended at time of cast, up to the character's current available mana. Regardless of the result of the damage portion of the spell, the caster may still cast the healing portion (ie, if the damage is resisted or missed, the caster may still cast the Magic Heal Wounds). |
 
 #### **Life**
```

Comment:
Moving Life to Rank 10 would be good to call out in the tl;dr section of the rework.